components/molecules/BenefitApplicationCard.js

```diff
@@ -37,7 +45,11 @@ const BenefitApplicationCard = (props) => {
               href={props.benefitApplication.learnMoreLink}
               className="text-xl underline hover:text-bright-blue-solid"
             >
-              {props.benefitApplication.benefitName}
+              {typeof props.benefitApplication.benefitSubType === 'undefined'
+                ? t.applyFor + ' ' + props.benefitApplication.benefitName
```

Comment:
maybe a function that takes 3 params and returns the string already formatted would be more efficient

Reply:
I added function to return string but I didn't add 3 parameters to use it for both "Apply" button and "Learn More" link.